src/contracts/facilitators/gsm/GhoRemoteVault.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {IERC20} from '@aave/core-v3/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
+import {AccessControl} from '@openzeppelin/contracts/access/AccessControl.sol';
+import {VersionedInitializable} from '@aave/core-v3/contracts/protocol/libraries/aave-upgradeability/VersionedInitializable.sol';
+import {IGhoRemoteVault} from './interfaces/IGhoRemoteVault.sol';
+
+contract GhoRemoteVault is IGhoRemoteVault, AccessControl, VersionedInitializable {
```

Comment:
```suggestion
import {IERC20} from '@aave/core-v3/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
import {VersionedInitializable} from '@aave/core-v3/contracts/protocol/libraries/aave-upgradeability/VersionedInitializable.sol';
import {IGhoRemoteVault} from './interfaces/IGhoRemoteVault.sol';

contract GhoRemoteVault is VersionedInitializable, IGhoRemoteVault {
```

Reply:
let's add natspec docs here. Let's rename to `GhoReserve`, do you think is clearer?